mix.exs

```diff
@@ -32,11 +32,10 @@ defmodule Elixometer.Mixfile do
 
   defp deps do
     [
-        {:meck, "~> 0.8.3"},
+        {:meck, "~> 0.8.3", only: :test},
         {:edown, github: "uwiger/edown", tag: "0.7", override: true},
         {:lager, github: "basho/lager", tag: "2.1.0", override: true},
-        {:exometer, github: "pspdfkit-labs/exometer"},
-        {:netlink, github: "Feuerlabs/netlink", ref: "d6e7188e", override: true},
+        {:exometer_core, git: "git://github.com/PSPDFKit-labs/exometer_core.git", branch: "master"},
```

Comment:
I did, but this caused a conflict when I used this in our internal project; The conflict stems from exometer's rebar file using git://, while `github` in mix.exs causes it to use `http` as the protocol.